README.md

```diff
@@ -35,6 +35,13 @@ Plugin options:
 | `release` | no | Unique identifier of the release. By default it will be Webpack's compilation hash. You **should** pass this identifier to the [Javascript Catcher](https://github.com/codex-team/hawk.javascript) on initialization through the `release` option |
 | `releaseInfoFile` | no | The path where `release.json` file will be created. By default, it will be got from Webpack `output.path` option. You can pass `false` to prevent creation of this file (can be useful, if you store and pass release id manually) |
 | `removeSourceMaps` | no | Should the plugin to remove emitted source map files. Default is `true`. |
+| `getCommits` | no | Should the plugin try to get commits from the repository. Default is `false` |
+| `gitOptions` | no | Options for getting the commits |
```

Comment:
two options for a single reason. It's better to leave one of them. For example:

```
{
   // ...
   commits: { // also can be false
     repo: '',
     count: 5
   }
 }
```

Reply:
It is better not to rename `number` to  `count`. These settings are identical in the module, if the user makes a mistake, he will get the correct error